Source/Scene/Cesium3DTileset.js

```diff
@@ -440,28 +397,31 @@ define([
             var childrenLength = children.length;
             var child;
             var k;
-            var additiveRefinment = (t.refine === Cesium3DTileRefine.ADD);
+            var additiveRefinement = (t.refine === Cesium3DTileRefine.ADD);
+
+            if (t.hasTilesetContent) {
+                // If tile has tileset content, skip it and process its child instead (the tileset root)
+                // No need to check visibility or sse of the child because its bounding volume
+                // and geometric error are equal to its parent.
+                if (t.isReady()) {
+                    child = t.children[0];
+                    if (child.isContentUnloaded()) {
+                        requestContent(tiles3D, child, outOfCore);
+                    } else {
+                        stack.push(child);
```

Comment:
@lilleyse The child tile pushed on the stack will have `distanceToCamera === 0` here, so when it comes off the stack `sse` will be calculated based on that. Don't we need to compute tile distance here?